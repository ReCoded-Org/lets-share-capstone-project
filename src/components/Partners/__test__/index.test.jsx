"use strict";

import renderer from "react-test-renderer";

import Index from "../index.js";

it("renders correctly", () => {
    const tree = renderer.create(<Index />).toJSON();
    expect(tree).toMatchSnapshot();
});
