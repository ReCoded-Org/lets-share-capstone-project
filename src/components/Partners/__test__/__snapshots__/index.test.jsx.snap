// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="pt-20 pb-14"
>
  <h2
    className="text-center font-head text-4xl font-bold tracking-wider text-primary"
  >
    common.partners
  </h2>
  <div
    className=" container mx-auto mt-10 flex grid-cols-2 flex-col items-center gap-10 px-4 text-center sm:grid md:grid-cols-4 md:px-10 lg:flex-row lg:justify-around xl:grid-cols-4"
  >
    <div
      className="relative mx-auto aspect-video w-64 md:w-40"
    >
      <span
        style={
          Object {
            "background": "none",
            "border": 0,
            "boxSizing": "border-box",
            "display": "block",
            "height": "initial",
            "margin": 0,
            "opacity": 1,
            "overflow": "hidden",
            "padding": 0,
            "position": "relative",
            "width": "initial",
          }
        }
      >
        <span
          style={
            Object {
              "background": "none",
              "border": 0,
              "boxSizing": "border-box",
              "display": "block",
              "height": "initial",
              "margin": 0,
              "opacity": 1,
              "padding": 0,
              "paddingTop": "60%",
              "width": "initial",
            }
          }
        />
        <img
          alt="partner"
          className="rounded-md"
          data-nimg="responsive"
          decoding="async"
          onError={[Function]}
          onLoad={[Function]}
          src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
          style={
            Object {
              "border": "none",
              "bottom": 0,
              "boxSizing": "border-box",
              "display": "block",
              "height": 0,
              "left": 0,
              "margin": "auto",
              "maxHeight": "100%",
              "maxWidth": "100%",
              "minHeight": "100%",
              "minWidth": "100%",
              "objectFit": undefined,
              "objectPosition": undefined,
              "padding": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": 0,
            }
          }
        />
        <noscript>
          <img
            alt="partner"
            className="rounded-md"
            data-nimg="responsive"
            decoding="async"
            loading="lazy"
            sizes="100vw"
            src="/_next/image?url=%2Fimages%2FReCoded.webp&w=3840&q=75"
            srcSet="/_next/image?url=%2Fimages%2FReCoded.webp&w=640&q=75 640w, /_next/image?url=%2Fimages%2FReCoded.webp&w=750&q=75 750w, /_next/image?url=%2Fimages%2FReCoded.webp&w=828&q=75 828w, /_next/image?url=%2Fimages%2FReCoded.webp&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2FReCoded.webp&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2FReCoded.webp&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2FReCoded.webp&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2FReCoded.webp&w=3840&q=75 3840w"
            style={
              Object {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
        </noscript>
      </span>
    </div>
    <div
      className="relative mx-auto aspect-video w-64 md:w-40"
    >
      <span
        style={
          Object {
            "background": "none",
            "border": 0,
            "boxSizing": "border-box",
            "display": "block",
            "height": "initial",
            "margin": 0,
            "opacity": 1,
            "overflow": "hidden",
            "padding": 0,
            "position": "relative",
            "width": "initial",
          }
        }
      >
        <span
          style={
            Object {
              "background": "none",
              "border": 0,
              "boxSizing": "border-box",
              "display": "block",
              "height": "initial",
              "margin": 0,
              "opacity": 1,
              "padding": 0,
              "paddingTop": "60%",
              "width": "initial",
            }
          }
        />
        <img
          alt="partner"
          className="rounded-md"
          data-nimg="responsive"
          decoding="async"
          onError={[Function]}
          onLoad={[Function]}
          src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
          style={
            Object {
              "border": "none",
              "bottom": 0,
              "boxSizing": "border-box",
              "display": "block",
              "height": 0,
              "left": 0,
              "margin": "auto",
              "maxHeight": "100%",
              "maxWidth": "100%",
              "minHeight": "100%",
              "minWidth": "100%",
              "objectFit": undefined,
              "objectPosition": undefined,
              "padding": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": 0,
            }
          }
        />
        <noscript>
          <img
            alt="partner"
            className="rounded-md"
            data-nimg="responsive"
            decoding="async"
            loading="lazy"
            sizes="100vw"
            src="/_next/image?url=%2Fimages%2FUS_Department.svg.png&w=3840&q=75"
            srcSet="/_next/image?url=%2Fimages%2FUS_Department.svg.png&w=640&q=75 640w, /_next/image?url=%2Fimages%2FUS_Department.svg.png&w=750&q=75 750w, /_next/image?url=%2Fimages%2FUS_Department.svg.png&w=828&q=75 828w, /_next/image?url=%2Fimages%2FUS_Department.svg.png&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2FUS_Department.svg.png&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2FUS_Department.svg.png&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2FUS_Department.svg.png&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2FUS_Department.svg.png&w=3840&q=75 3840w"
            style={
              Object {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
        </noscript>
      </span>
    </div>
    <div
      className="relative mx-auto aspect-video w-64 md:w-40"
    >
      <span
        style={
          Object {
            "background": "none",
            "border": 0,
            "boxSizing": "border-box",
            "display": "block",
            "height": "initial",
            "margin": 0,
            "opacity": 1,
            "overflow": "hidden",
            "padding": 0,
            "position": "relative",
            "width": "initial",
          }
        }
      >
        <span
          style={
            Object {
              "background": "none",
              "border": 0,
              "boxSizing": "border-box",
              "display": "block",
              "height": "initial",
              "margin": 0,
              "opacity": 1,
              "padding": 0,
              "paddingTop": "60%",
              "width": "initial",
            }
          }
        />
        <img
          alt="partner"
          className="rounded-md"
          data-nimg="responsive"
          decoding="async"
          onError={[Function]}
          onLoad={[Function]}
          src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
          style={
            Object {
              "border": "none",
              "bottom": 0,
              "boxSizing": "border-box",
              "display": "block",
              "height": 0,
              "left": 0,
              "margin": "auto",
              "maxHeight": "100%",
              "maxWidth": "100%",
              "minHeight": "100%",
              "minWidth": "100%",
              "objectFit": undefined,
              "objectPosition": undefined,
              "padding": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": 0,
            }
          }
        />
        <noscript>
          <img
            alt="partner"
            className="rounded-md"
            data-nimg="responsive"
            decoding="async"
            loading="lazy"
            sizes="100vw"
            src="/_next/image?url=%2Fimages%2FQatarCharity.png&w=3840&q=75"
            srcSet="/_next/image?url=%2Fimages%2FQatarCharity.png&w=640&q=75 640w, /_next/image?url=%2Fimages%2FQatarCharity.png&w=750&q=75 750w, /_next/image?url=%2Fimages%2FQatarCharity.png&w=828&q=75 828w, /_next/image?url=%2Fimages%2FQatarCharity.png&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2FQatarCharity.png&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2FQatarCharity.png&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2FQatarCharity.png&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2FQatarCharity.png&w=3840&q=75 3840w"
            style={
              Object {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
        </noscript>
      </span>
    </div>
    <div
      className="relative mx-auto aspect-video w-64 md:w-40"
    >
      <span
        style={
          Object {
            "background": "none",
            "border": 0,
            "boxSizing": "border-box",
            "display": "block",
            "height": "initial",
            "margin": 0,
            "opacity": 1,
            "overflow": "hidden",
            "padding": 0,
            "position": "relative",
            "width": "initial",
          }
        }
      >
        <span
          style={
            Object {
              "background": "none",
              "border": 0,
              "boxSizing": "border-box",
              "display": "block",
              "height": "initial",
              "margin": 0,
              "opacity": 1,
              "padding": 0,
              "paddingTop": "60%",
              "width": "initial",
            }
          }
        />
        <img
          alt="partner"
          className="rounded-md"
          data-nimg="responsive"
          decoding="async"
          onError={[Function]}
          onLoad={[Function]}
          src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
          style={
            Object {
              "border": "none",
              "bottom": 0,
              "boxSizing": "border-box",
              "display": "block",
              "height": 0,
              "left": 0,
              "margin": "auto",
              "maxHeight": "100%",
              "maxWidth": "100%",
              "minHeight": "100%",
              "minWidth": "100%",
              "objectFit": undefined,
              "objectPosition": undefined,
              "padding": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": 0,
            }
          }
        />
        <noscript>
          <img
            alt="partner"
            className="rounded-md"
            data-nimg="responsive"
            decoding="async"
            loading="lazy"
            sizes="100vw"
            src="/_next/image?url=%2Fimages%2FFlatiron_School.png&w=3840&q=75"
            srcSet="/_next/image?url=%2Fimages%2FFlatiron_School.png&w=640&q=75 640w, /_next/image?url=%2Fimages%2FFlatiron_School.png&w=750&q=75 750w, /_next/image?url=%2Fimages%2FFlatiron_School.png&w=828&q=75 828w, /_next/image?url=%2Fimages%2FFlatiron_School.png&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2FFlatiron_School.png&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2FFlatiron_School.png&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2FFlatiron_School.png&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2FFlatiron_School.png&w=3840&q=75 3840w"
            style={
              Object {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
        </noscript>
      </span>
    </div>
  </div>
</div>
`;
